### ISSUE TITLE
Example results

### ISSUE THREAD

**Issue by fabvio** [ali-vilab/Ranni#10]
Hi, I tried to reproduce some of your examples and this is what I got:

<img width="1337" alt="Screenshot 2024-04-10 at 13 10 07" src="https://github.com/ali-vilab/Ranni/assets/10325202/868d47e1-1172-46ca-af1c-b4c15ad5c0c3">

While I understand that errors may occur, it looks to me that something is broken in my installation, am I correct? I have loaded LLaMA in `bfloat16`, could it be caused by that in your opinion?

**Comment by thss15fyt** [comment]
Yeah. It looks like the text-to-panel results are different with the same seed, which might be caused by the Llama dtype.
